ComfortZoneApp/app/src/main/java/com/example/comfortzone/models/WeatherData.java

```diff
@@ -26,4 +33,26 @@ public double getTemp() {
             return temp;
         }
     }
+
+    public void setDate() {
+        DateTimeFormatter dtf = DateTimeFormatter.ofLocalizedDate(FormatStyle.SHORT);
+        LocalDateTime now = LocalDateTime.now();
+        String date = now.format(dtf);
+        this.date = date;
+    }
+
+    public void setTime() {
+        DateTimeFormatter dtf = DateTimeFormatter.ofLocalizedTime(FormatStyle.SHORT);
+        LocalDateTime now = LocalDateTime.now();
+        String time = now.format(dtf);
+        this.time = time;
+    }
```

Comment:
Do they respect system time zone?

Just suggestions, but these are more commonly used in Android:
- I think you can just use `Date` object to represent time without separating them: https://developer.android.com/reference/java/util/Date
- to get `now`, you can just get an instance of the `Calendar` then use `getTime()` to get `Date` object: https://developer.android.com/reference/java/util/Calendar#getTime()
- for the format, you can use `SimpleDateFormat` to customize more: https://developer.android.com/reference/java/text/SimpleDateFormat

maybe you can just use one method to "set time" using these